worlds/animal_well/names.py

```diff
@@ -463,3 +464,120 @@ def __str__(self) -> str:
+    fruit_18 = "Pink Fruit by Barcode Bunny - R3C5"
+    fruit_19 = "Blue Fruit in Dog Switch Maze - R3C7"  # blue fruit
+    fruit_20 = "Pink Fruit Right of Dog Switch Maze Left - R3C8a"
+    fruit_21 = "Pink Fruit Right of Dog Switch Maze Right - R3C8b"
+    fruit_22 = "Pink Fruit in Dog Elevator Below P. Flame - R3C10"
+    fruit_23 = "Pink Fruit in Bear Middle Phone Room - R3C12"
+    fruit_24 = "Pink Fruit by Monkey - R3C16"
+    fruit_25 = "Blue Fruit in Kangaroo Room Hand - R4C1a"  # blue fruit upper
+    fruit_26 = "Blue Fruit in Kangaroo Room Tail - R4C1b"  # blue fruit lower
+    fruit_27 = "Pink Fruit in Kangaroo Room Foot - R4C1c"
+    fruit_28 = "Pink Fruit by Turtle Flute Pool - R4C6"
+    fruit_29 = "Pink Fruit Left of Doodle Bunny - R4C11"
+    fruit_30 = "Pink Fruit by Statue Bunny - R4C13"
+    fruit_31 = "Blue Fruit in Capybara Room Top Left - R4C15a"  # blue fruit top left
+    fruit_32 = "Blue Fruit in Capybara Room Top Right - R4C15b"  # blue fruit top right
+    fruit_33 = "Pink Fruit in Capybara Room Bottom Right - R4C15c"  # pink fruit bottom right
+    fruit_34 = "Pink Fruit in Capybara Room Bottom Left - R4C15d"  # pink fruit bottom left
+    fruit_35 = "Blue Fruit Above Plant Egg Chest - R5C1"  # blue fruit
+    fruit_36 = "Pink Fruit Below Dog Many Switches Room - R5C3"
+    fruit_37 = "Pink Fruit Before Mock Disc Chest - R5C4"
+    fruit_38 = "Pink Fruit by Phone Left of Mock Disc Chest - R5C6"
+    fruit_39 = "Blue Fruit by Dog Wheel - R5C9"  # blue fruit
+    fruit_40 = "Blue Fruit Above Bear Kangaroo Waterfall - R5C11"  # blue fruit
+    fruit_41 = "Blue Fruit by Universal Basic Egg Chest - R5C13"  # blue fruit
+    fruit_42 = "Pink Fruit in Dog Dark Room - R6C5"
+    fruit_43 = "Blue Fruit After Dachshund Tunnels - R6C9a"  # blue fruit
+    fruit_44 = "Pink Fruit Above Egg Room - R6C9b"
+    fruit_45 = "Pink Fruit Before Truth Egg - R6C10a"
+    fruit_46 = "Pink Fruit Bottom of Dog Elevator - R6C10b"
+    fruit_47 = "Pink Fruit by Bear Kangaroo Waterfall - R6C11"
+    fruit_48 = "Blue Fruit After First Chameleon Fight - R6C12"  # blue fruit
+    fruit_49 = "Blue Fruit in Bear Dark Maze Right - R6C14a"  # blue fruit, verified
+    fruit_50 = "Pink Fruit in Bear Dark Maze Left - R6C14b"
+    fruit_51 = "Pink Fruit Below Capybara Room - R6C15"
+    fruit_52 = "Pink Fruit in Chinchilla Vines Room - R6C16"
+    fruit_53 = "Blue Fruit by Ancient Egg Chest Right - R7C1a"  # blue fruit, verified
+    fruit_54 = "Pink Fruit by Ancient Egg Chest Left - R7C1b"
+    fruit_55 = "Pink Fruit in Fish Pipe Maze Left - R7C2"
+    fruit_56 = "Blue Fruit in Fish Pipe Maze Right - R7C3a"  # blue fruit, verified
+    fruit_57 = "Pink Fruit in Fish Pipe Maze Middle - R7C3b"
+    fruit_58 = "Pink Fruit by Fireball Thrower - R7C4"
+    fruit_59 = "Pink Fruit by Flute Chest - R7C6"
+    fruit_60 = "Blue Fruit Behind Entry Chameleon - R7C14"  # blue fruit
+    fruit_61 = "Pink Fruit in Bear Entry Room - R7C15"
```

Comment:
I feel like we've already been in bear for a few rooms by this point. Maybe this room needs a better name?

Reply:
```suggestion
    fruit_61 = "Pink Fruit left of Lower Bear Key Room - R7C15"
```